### PR TITLE
add fluidfft with pythran/fftw/mpi4py

### PR DESCRIPTION
**Pull request by CyrilleBonamy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by paugier** [comment]
I confirm!

**Comment by paugier** [comment]
@CyrilleBonamy It's strange that you removed the transonic recipe in this PR...

File: recipes/transonic/meta.yaml [hunk 1]
```diff
@@ -1,59 +0,0 @@
-{% set name = "transonic" %}
-{% set version = "0.2.1" %}
-
```
Comment (marcelotrevisani) [inline recipes/transonic/meta.yaml]: please do not touch it,

**Comment by CyrilleBonamy** [comment]
> @CyrilleBonamy It's strange that you removed the transonic recipe in this PR...

it is because of a small error in the creation of the fluidfft branch : I left the transonic branch and not the master ...

File: recipes/transonic/meta.yaml (hunk 1 above)
Comment (CyrilleBonamy) [reply recipes/transonic/meta.yaml]: i am not sure what to do... So i close this pull request and try again :)
Thanks @marcelotrevisani for the blas correction

**Pull request closed without merging**